code/modules/mob/living/carbon/human/species_types.dm

```diff
@@ -82,9 +82,9 @@ datum/species/human/spec_death(gibbed, mob/living/carbon/human/H)
 	return 1
 
 /datum/species/lizard/handle_speech(message)
-	// jesus christ why
+
 	if(copytext(message, 1, 2) != "*")
-		message = replacetext(message, "s", "sss")
+		message = regex_replaceall(message, "(?<!s)s(?!s)", "sss") //(?<!s) Not s before. (?!s) not s after. That way it only triples a single s instead of double ss.
```

Comment:
but can your precious regex make it so upper case `S` is followed by other uppercase `S`'s